### PR TITLE
ISPN-5359 Drop old (<1.4) server parsers

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-5359

### CODE REVIEW

File: server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanExtension.java [hunk 1]
```diff
@@ -81,9 +81,6 @@ public void initialize(ExtensionContext context) {
 
         subsystem.registerSubsystemModel(new InfinispanSubsystemRootResource(resolvePathHandler, context.isRuntimeOnlyRegistrationValid()));
         subsystem.registerXMLElementWriter(new InfinispanSubsystemXMLWriter());
-        if (context.isRegisterTransformers()) {
```
Comment (pruivo) [inline server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanExtension.java]: I don't understand much of the server parser, but this changes does not make sense to me in this context... except if the transformers are only used in the old parsers (<1.4)

Comment (tristantarrant) [inline server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanExtension.java]: Yes, before 1.4 the schema didn't allow expressions in place of values (e.g. ${...}). The transformers were put in place to be able to handle that when translating configurations between the various versions (e.g. newer domain controller with older hosts)

**Comment by pruivo** [comment]
testing and pushing...

**Comment by pruivo** [comment]
integrated! thanks @tristantarrant !

**Pull request closed without merging**